pkg/storageops/vsphere/vsphere.go

```diff
@@ -0,0 +1,376 @@
+package vsphere
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/kubernetes/kubernetes/pkg/cloudprovider/providers/vsphere/vclib/diskmanagers"
```

Comment:
nit: we could just stick to either github.com/kubernetes or k8s.io/kubernetes (line no. 14)

Reply:
done